trade-finance-manager-ui/component-tests/utilisation-reports/utilisation-report-reconciliation-for-report.component-test.ts

```diff
@@ -97,6 +97,10 @@ describe(page, () => {
           /When payments show as matched, the adjustment data for keying into ACBS will be automatically generated when the 'Generate keying data' button is selected./,
         );
 
+      wrapper
+        .expectText(`[data-cy="how-to-create-record-correction-request-text"]`)
+        .toMatch(/If there is an error with the fee record, select the record to create a record correction request to send a query to the bank./);
```

Comment:
Should we be using `.toRead` here and below, instead of `.toMatch`?

Is `toMatch` stricter? If so please ignore

Reply:
changed to toRead